tests/transform_clamp/generateTest.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+##
+# SPDX-License-Identifier: LGPL-2.1-only
```

Comment:
SPDX license should be mentioned on the second line of the file

Reply:
Well, in nnstreamer, there're many python scripts with the same description

Reply:
Then thats not correct and must be fixed.
Refernce - https://www.kernel.org/doc/Documentation/process/license-rules.rst

Reply:
I think that's the rule in kernel scripts. Also, I'm not sure that's out-of-scope in this PR. If it's really mandatory, fixing license issues across all python scripts can be handled in another PR.

Reply:
Yes, fixing all the scripts is out of scope for this PR. However, the license for the files added in the PR can be fixed to a proper style.

Reply:
I don't want to fix this because I'm not sure this is really necessary. I will keep the same format with other source files in nnstreamer. Also, I think that's not a hard rule. There are many python source files with SPDX specifier, available in many open source repo.

Reply:
ok